modules/hostap/src/supp_api.c

```diff
@@ -365,6 +365,15 @@ static int wpas_add_and_config_network(struct wpa_supplicant *wpa_s,
 				_wpa_cli_cmd_v("set_network %d proto WPA",
 					resp.network_id);
 			}
+		} else if (params->security == WIFI_SECURITY_TYPE_WPA_AUTO_PERSONAL) {
+			_wpa_cli_cmd_v("set_network %d sae_password \"%s\"",
```

Comment:
This needs to be protected with `if sae_password`

Reply:
Done